packages/reports/addon/components/report-builder.js

```diff
@@ -39,5 +40,55 @@ export default Component.extend({
   allTables: computed(function() {
     let metadataService = get(this, 'metadataService');
     return metadataService.all('table').sortBy('longName');
-  })
+  }),
+
+  actions: {
+    /**
+     * @action expandFilters
+     * @param {Function} shouldExpand
+     */
+    expandFilters(shouldExpand) {
```

Comment:
It doesn't look like `expandFilters` is being invoked from a template anywhere. Does it need to be an action?